src/components/dataset/settings.vue

```diff
@@ -0,0 +1,133 @@
+<!--
+Copyright 2023 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div id="dataset-settings">
+    <div v-if="dataset.dataExists" class="panel panel-simple">
+      <div class="panel-heading">
+        <h1 class="panel-title">{{ $t('entityWorkflow') }}</h1>
+      </div>
+      <div class="panel-body">
+        <form id="dataset-settings-form">
+          <div class="radio">
+            <label>
+              <input v-model="approvalRequired" name="approvalRequired" type="radio" :value="false"
+                aria-describedby="dataset-setting-on-receipt" :aria-disabled="dataset.awaitingResponse" @change="update()">
+              <strong>{{ $t('onReceipt.label') }}</strong>
+            </label>
+            <p id="dataset-setting-on-receipt" class="help-block">
+              {{ $t('onReceipt.description') }}
+            </p>
+          </div>
+          <div class="radio">
+            <label>
+              <input v-model="approvalRequired" name="approvalRequired" type="radio" :value="true"
```

Comment:
I'm actually not sure whether `aria-disabled` works here or if it should be `disabled`. This is our first disabled radio input, so you are breaking new ground. For most inputs, it's `aria-disabled`, but in this case, I think it should probably be `disabled`. `disabled` is what's needed for checkboxes: see the comments about checkboxes in src/composables/disabled.js. Some of those comments are relevant to radio inputs as well. In app.scss, it looks like we style radio inputs based on `disabled`, not `aria-disabled`.